src/main/java/org/tron/core/db/Manager.java

```diff
@@ -207,6 +218,12 @@
   @Getter
   private ForkController forkController = ForkController.instance();
 
+  private ExecutorService executorService;
+
+  public static Map<ByteArrayWrapper, AccountCapsule> accountCache = new HashMap<>();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this member "protected". ([squid:S2386](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2386))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1916&issues=AWgnKDa4rt4Kct_4Nf2F&open=AWgnKDa4rt4Kct_4Nf2F)

[ISSUE_KEY]: AWgnKDa4rt4Kct_4Nf2F